src/platform/unix/mod.rs

```diff
@@ -863,6 +861,7 @@ impl Deref for OsIpcSharedMemory {
 }
 
 impl OsIpcSharedMemory {
+    /// # Safety
```

Comment:
```suggestion
    /// # Safety
    ///
    /// This is safe if there is only one reader/writer on the data.
    /// User can achieve this by not cloning [`IpcSharedMemory`]
    /// and serializing/deserializing only once.
```

from https://docs.rs/ipc-channel/latest/ipc_channel/ipc/struct.IpcSharedMemory.html#method.deref_mut